minard/views.py

```diff
@@ -280,6 +282,19 @@ def snostream():
 def nhit():
   return render_template('nhit.html')
 
+@app.route('/rat')
+def rathome():
+    return render_template('rathome.html', runs=nlrat.available_runs())
+    
+@app.route('/rat/<int:run>')
+def ratrun(run = 0):
+    return render_template("ratrun.html", run=nlrat.Run(run), error= not nlrat.hists_available(run))
+
+@app.route('/rat/<filename>')
+def get_root_file(filename):
```

Comment:
This is better served using the nginx server. I should be able to set this up as soon as we set up the NFS mount between nlug and minard.